src/pages/NewChatConfirmPage.tsx

```diff
@@ -0,0 +1,152 @@
+    const participantAccountIDs = newGroupDraft?.participants.map((participant) => participant.accountID);
+    const selectedOptions = useMemo((): Participant[] => {
+        if (!newGroupDraft?.participants) {
+            return [];
+        }
+        const options: Participant[] = newGroupDraft.participants.map((participant) =>
+            OptionsListUtils.getParticipantsOption({accountID: participant.accountID, login: participant.login, reportID: ''}, allPersonalDetails),
+        );
+        return options;
+    }, [allPersonalDetails, newGroupDraft?.participants]);
+
+    const groupName = ReportUtils.getGroupChatName(participantAccountIDs ?? []);
+
+    const sections: ListItem[] = useMemo(
+        () =>
+            selectedOptions
+                .map((selectedOption: Participant) => {
+                    const accountID = selectedOption.accountID;
+                    const isAdmin = personalData.accountID === accountID;
+                    let roleBadge = null;
+                    if (isAdmin) {
+                        roleBadge = (
+                            <Badge
+                                text={translate('common.admin')}
+                                textStyles={styles.textStrong}
+                                badgeStyles={[styles.justifyContentCenter, StyleUtils.getMinimumWidth(60), styles.badgeBordered, styles.activeItemBadge]}
+                            />
+                        );
+                    }
+
+                    const section: ListItem = {
+                        login: selectedOption?.login ?? '',
+                        text: selectedOption?.text ?? '',
+                        keyForList: selectedOption?.keyForList ?? '',
+                        isSelected: !isAdmin,
+                        isDisabled: isAdmin,
+                        rightElement: roleBadge,
+                        accountID,
+                        icons: selectedOption?.icons,
+                    };
+                    return section;
+                })
+                .sort((a, b) => a.text?.toLowerCase().localeCompare(b.text?.toLowerCase() ?? '') ?? -1),
+        [selectedOptions, personalData.accountID, translate, styles.textStrong, styles.justifyContentCenter, styles.badgeBordered, styles.activeItemBadge, StyleUtils],
+    );
+
+    /**
+     * Removes a selected option from list if already selected.
+     */
+    const unselectOption = (option: ListItem) => {
+        if (!newGroupDraft) {
+            return;
+        }
+        const newSelectedParticipants = newGroupDraft.participants.filter((participant) => participant.login !== option.login);
+        Report.setGroupDraft(newSelectedParticipants);
+    };
+
+    const createGroup = () => {
+        if (!newGroupDraft) {
+            return;
+        }
+        const logins: string[] = newGroupDraft.participants.map((participant) => participant.login);
+        Report.navigateToAndOpenReport(logins, true, groupName);
```

Comment:
According to the comment in `navigateToAndOpenReport`, logins should not include your own login which we currently include. To fix this:
1. In `NewChatPage` when calling `Report.setGroupDraft` do not include the current user
2. In `NewChatConfirmPage` section add current user to selectedOptions
3. In `navigateToAndOpenReport` make sure that calls to `buildOptimisticChatReport` include the current user (same way done with IOU)

Reply:
Hmm I'm curious if this leads to a bug or something? I'm not sure which case this would be a problem. Can you clarify @s77rt? Is this more of a refactor suggestion? Thanks!

Reply:
>According to the comment in navigateToAndOpenReport

Also not sure which comment we are referring to here. But in general the `participantAccountIDs` should:

1:1 DM chats - exclude current user
Other chats - exclude no one. the current user will be in the array.

Reply:
Ah the comment is what should updated be then 😅 

https://github.com/Expensify/App/blob/e542ba7c2d9e216c2cd401de0de8ed225fff4a7b/src/libs/actions/Report.ts#L747-L757